specs/010-reduce-coupling/data-model.md

```
@@ -0,0 +1,252 @@
+
+**Purpose**: Groups all compression concerns. Owns the `CompressionQueueManager` instance (no longer a singleton).
+
+```swift
+struct CompressionModuleGroup {
+    let facade: CompressionFacade          // Primary API for consumers
+    let queueManager: CompressionQueueManager  // Exposed for compression queue UI view
+}
+```
+
+**Initialization Order**:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)